elasticsearch.md

```diff
@@ -1124,13 +1124,45 @@ Local ReadonlyREST users are authenticated via HTTP Basic Auth. This authenticat
 
 Accepts [HTTP Basic Auth](https://en.wikipedia.org/wiki/Basic_access_authentication). Configure this value _in clear text_. Clients will need to provide the header e.g. `Authorization: Basic c2FsZXM6cDQ1NXdk` where "c2FsZXM6cDQ1NXdk" is Base64 for "sales:p455wd".
 
-**⚠️IMPORTANT**: this rule is handy just for tests, replace it with another rule that hashes credentials, like: `auth_key_sha256`, or `auth_key_unix`.
+**⚠️IMPORTANT**: this rule is handy just for tests, replace it with another rule that hashes credentials, like: `auth_key_sha512`, or `auth_key_unix`.
 
-#### `auth_key_sha256`
+#### `auth_key_sha512`
 
-`auth_key_sha256: 280ac6f...94bf9`
+`auth_key_sha512: 280ac6f...94bf9`
 
-Accepts [HTTP Basic Auth](https://en.wikipedia.org/wiki/Basic_access_authentication). The value is a string like `username:password` _hashed in_ [_SHA256_](http://www.xorbin.com/tools/sha256-hash-calculator). Clients will need to provide the usual Authorization header. \|
+Accepts [HTTP Basic Auth](https://en.wikipedia.org/wiki/Basic_access_authentication). The value is a string like `username:password` _hashed in_ [_SHA512_](https://md5calc.com/hash/sha512). Clients will need to provide the usual Authorization header. 
+
+There are also available other rules with less secure SHA algorithms `auth_key_sha256` and `auth_key_sha1`.
+
+The rules support also alternative syntax, where only password is hashed, eg:
+
+`auth_key_sha512: "admin:280ac6f...94bf9"`
+
+In the example below `admin` is a username and `280ac6f...94bf9` is
+hashed secret.
```

Comment:
in the example above, `admin` is the username, and `...` is the hashed password.